src-java/base-topology/base-messaging/src/main/java/org/openkilda/messaging/info/network/PathValidationResult.java

```diff
@@ -0,0 +1,46 @@
+/* Copyright 2018 Telstra Open Source
+ *
+ *   Licensed under the Apache License, Version 2.0 (the "License");
+ *   you may not use this file except in compliance with the License.
+ *   You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   Unless required by applicable law or agreed to in writing, software
+ *   distributed under the License is distributed on an "AS IS" BASIS,
+ *   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *   See the License for the specific language governing permissions and
+ *   limitations under the License.
+ */
+
+package org.openkilda.messaging.info.network;
+
+import org.openkilda.messaging.info.InfoData;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.databind.PropertyNamingStrategy;
+import com.fasterxml.jackson.databind.annotation.JsonNaming;
+import lombok.Builder;
+import lombok.EqualsAndHashCode;
+import lombok.Value;
+
+import java.util.List;
+
+@Value
+@Builder
+@EqualsAndHashCode(callSuper = false)
+@JsonNaming(value = PropertyNamingStrategy.SnakeCaseStrategy.class)
+public class PathValidationResult extends InfoData {
+    @JsonProperty("is_valid")
+    Boolean isValid;
```

Comment:
In the end they are private. Lombok preprocessor sees the Value annotation and changes this class to be an immutable class with getters for all fields. It also makes all fields private. Yeah, not sure I like it as well, maybe it's better to be explicit and disable the corresponding inspection in idea (it will mark private as redundant).

Reply:
I see. Thank you for the clarification!